SPECS-SIGNED/fwctl-signed/fwctl-signed.spec

```diff
@@ -0,0 +1,142 @@
+#
```

Comment:
I've added a bunch of comments into this spec but they apply to most/all other signed spec files.

Reply:
Fixed for all signed SPECS.